portal-frontend/src/app/features/notice-of-intents/edit-submission/proposal/pofo/pofo-proposal.component.ts

```diff
@@ -154,11 +153,6 @@ export class PofoProposalComponent extends FilesStepComponent implements OnInit,
         soilAlreadyPlacedVolume: this.fillAlreadyPlacedTableData?.volume ?? null,
         soilAlreadyPlacedArea: this.fillAlreadyPlacedTableData?.area ?? null,
         soilAlreadyPlacedMaximumDepth: this.fillAlreadyPlacedTableData?.maximumDepth ?? null,
-        soilAlreadyPlacedAverageDepth: this.fillAlreadyPlacedTableData?.averageDepth ?? null,
```

Comment:
Was there a reason `soilAlreadyPlacedAverageDepth` was deleted or was it an accident?

Reply:
was an accident